src/tools/segmentation/index.js

```diff
@@ -0,0 +1,24 @@
+import BrushTool from './BrushTool';
```

Comment:
Hmm... So if we change all of these to extend `BaseTool.js`, it may still be worth grouping them in a directory.